nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -147,6 +147,8 @@
 - [chromadb](https://www.trychroma.com/), an open-source AI application
   database. Batteries included. Available as [services.chromadb](options.html#opt-services.chromadb.enable).
 
+- [vivid](https://github.com/sharkdp/vivid), a generator for LS_COLOR. Available as [programs.vivid](#opt-programs.vivid.enable).
```

Comment:
Should go in 25.05 unless you want to have it backported.

Reply:
Nice catch! Thanks, fixed.